src/main/java/pl/tomaszdziurko/jvm_bloggers/UTMParametrsConstants.java

```diff
@@ -0,0 +1,25 @@
+package pl.tomaszdziurko.jvm_bloggers;
+
+/**
+ * UTM (Urchin Traffic Monitor) URL query parameter identifiers.
+ * 
+ * @author cslysy <jakub.sprega@gmail.com>
+ */
+public class UTMParametrsConstants {
```

Comment:
Ok, I understand your point of view, I will not insist on that. Let's @tdziurko make the final decision then. But at least make the class `abstract` and correct a typo in
_UTMParametrsConstants_ ->  _UTMParamet**e**rsConstants_. I'm also thinking about strippping _Constants_ prefix and leave only `UTMParameters` what do you think?